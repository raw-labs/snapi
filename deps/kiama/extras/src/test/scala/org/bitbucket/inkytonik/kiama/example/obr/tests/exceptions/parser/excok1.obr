(* Legal exception declaration *)

PROGRAM ExcOk1 (i : INTEGER) : INTEGER;

CONST
   CountException : EXCEPTION;

BEGIN
END ExcOk1.
