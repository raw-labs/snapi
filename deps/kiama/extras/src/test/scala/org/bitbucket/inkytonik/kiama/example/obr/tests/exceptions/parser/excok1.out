ObrInt(ExcOk1,Vector(IntParam(IdnDef(i)), ExnConst(IdnDef(CountException))),Vector(),ExcOk1)
